src/libraries/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/TestService.cs

```diff
@@ -43,6 +43,10 @@ protected override void OnContinue()
         protected override void OnCustomCommand(int command)
         {
             base.OnCustomCommand(command);
+
+            if (Environment.UserInteractive) // verify this is false
```

Comment:
Is this test going to work on Windows Nano?

Reply:
Good point - I still had it backwards in my head. OK, I gave it another shot. I can't get these tests to pass currently anyway.
https://github.com/dotnet/runtime/pull/1234